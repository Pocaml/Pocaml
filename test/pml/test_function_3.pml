let f = function
    true -> "true"
  | _ -> "false"

let _ = print_string "f true = "; print_endline (f true)
let _ = print_string "f others = "; print_endline (f false)
