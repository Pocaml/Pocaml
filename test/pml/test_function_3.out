f true = true
f others = false
